packages/api/tests/endpoints/test-rules.js

```diff
@@ -745,39 +752,19 @@ test('PUT replaces a rule', async (t) => {
     .send(updateRule)
     .expect(200);
 
-  const actualRule = await ruleModel.get({ name: updateRule.name });
-  const actualPostgresRule = await t.context.rulePgModel
-    .get(t.context.testKnex, { name: updateRule.name });
-  const updatedEsRecord = await t.context.esRulesClient.get(
-    originalDynamoRule.name
-  );
+  const actualPostgresRule = await rulePgModel.get(testKnex, { name: updateRule.name });
+  const updatedEsRecord = await esRulesClient.get(translatedPgRecord.name);
 
-  t.true(actualRule.updatedAt > updateRule.updatedAt);
```

Comment:
I think we want to keep this as:

`t.true(actualPostgresRule.updated_at > originalPgRecord.updated_at);`

Reply:
[`4c12492` (#2871)](https://github.com/nasa/cumulus/pull/2871/commits/4c1249251939ede5e8b8d6fe90b051ca72effc6d#diff-49139a123843cc0118f2bc2b3275724314181f1de87be662e46c5125aca6c844R758)